src/redux/migration.ts

```diff
@@ -16,6 +16,13 @@ export const migrateEachYear = <S extends USTState>(state: S): S =>
       }
     })
 
+    // Ensure interestIncome on K1s
+    acc[year].scheduleK1Form1065s.forEach((k1) => {
```

Comment:
I didn't notice it in the w2s above, but there is state modification in place happening here, which is not good. It might be ok at this stage, since this is all only happening at first load, but in general in redux it leads to errors in rerendering caused by subscribers not being updated.  The issue is basically this:

```js
const ks = ['root']

const oldState = {
  root: [{ child: 'child1' }]
}

const newState = ks.reduce((acc, key) => {
  acc[key].forEach((item) => {
    item.child += ' hello'
  })
  return {
    ...acc
  }
}, oldState)

newState == oldState
// false -- Good, if a subscriber is looking at the root state, it rerenders

newState.root === oldState.root
// true -- Bad, a subscriber looking at the child of the state does not rerender

```

Reply:
Hm, okay. I assumed it was okay here since the intent is to migrate data. I don't have much experience with redux and was going off https://github.com/ustaxes/UsTaxes/pull/955#discussion_r819250361 and the surrounding code.

Reply:
Yea I agree, if we need to fix it later will do all cases in a different pr. Thanks!